### DISCUSSION TITLE
"No data" in grafana

### DISCUSSION THREAD

**Discussion by rackilucky** [teslamate-org/teslamate#2537] [Getting started]
Hi all,
I was using Teslamate 2.5 years with my old Model X and it worked perfectly fine.
For my new Model Y i set up the new version from scratch, edited the docker-compose.yml as described and logged in to Teslamate.

Teslamate itself works fine, I can see the main screen, my car, the actual status, Geo-Fences, everything ok ...
But when I try to access any of the available dashboard, all I can see is "No data" - no matter which one I choose.

As I did everything exactly as stated in the description I have no idea what went wrong or is wrong. 
Please help me :(

![image](https://user-images.githubusercontent.com/81877394/158031060-61290d27-76be-456d-9a17-594e4609b979.png)

**Comment by duncansmart** [comment]
Looks like this was [resolved](https://github.com/adriankumpf/teslamate/discussions/2540#discussioncomment-2356612)
